zkevm-circuits/src/evm_circuit/execution/end_block.rs

```diff
@@ -131,6 +161,27 @@ impl<F: Field> ExecutionGadget<F> for EndBlockGadget<F> {
         self.total_txs_is_max_txs
             .assign(region, offset, total_txs, max_txs)?;
         let max_txs_assigned = self.max_txs.assign(region, offset, Value::known(max_txs))?;
+
+        let withdraw_root = self.phase2_withdraw_root.assign(
+            region,
+            offset,
+            region.word_rlc(block.withdraw_root),
+        )?;
+        let _withdraw_root_prev = self.phase2_withdraw_root_prev.assign(
+            region,
+            offset,
+            region.word_rlc(block.prev_withdraw_root),
+        )?;
+
+        self.withdraw_root_assigned
+            .borrow_mut()
+            .replace(withdraw_root.cell());
+        // TODO: now we do not export withdraw_root_prev for we have only one
+        // phase2 cell which is enabled for copy constraint
+        // self.withdraw_root_prev_assigned
+        //     .borrow_mut()
+        //     .replace(withdraw_root_prev.cell());
+
```

Comment:
is this still relevant? @noel2004

Reply:
Not any more. Current state should be ok